onsite/global_flr.py

```
@@ -0,0 +1,717 @@
+                'site_count': 'Cumulative_Site_Count',
+                'flr': 'Cumulative_FLR',
+                'phospho': 'Cumulative_Phospho',
+                'phosphodecoy': 'Cumulative_PhosphoDecoy'
+            })
+            excel_path = output_path.replace('.csv', '.xlsx')
+            df.to_excel(excel_path, index=False)
+            logger.info(f"FLR data exported to: {excel_path}")
+        except ImportError:
+            pass  # openpyxl not available
+
+    def update_idxml_files(self, idxml_paths: List[str]) -> None:
```

Ruff (0.14.13)

Unused method argument: `idxml_paths`

(ARG002)